v19.2/architecture/life-of-a-distributed-transaction.md

```diff
@@ -170,20 +171,23 @@ Once the command achieves consensus (i.e., a majority of nodes including itself
 
-Note that this write operation creates a write intent; these writes will not be fully committed until the gateway node sets the transaction record's status to `COMMITTED`.
+Note that this write operation creates a write intent; these writes will not be fully committed until the gateway node sets the transaction record's status to one of the following states:
+
+- `COMMITTED`, which indicates that write intents can be treated as committed values.
+- `STAGING`, which is used to enable the [Parallel Commits](#parallel-commits) feature.  Depending on the state of the write intents referenced by this record, the transaction may or may not be in a committed state.
 
 ## On the way back up
 
 Now that we have followed an operation all the way down from the SQL client to RocksDB, we can pretty quickly cover what happens on the way back up (i.e., when generating a response to the client).
 
 1. Once the leaseholder applies a write to its Raft log,
- it sends an commit acknowledgment to the gateway node's `DistSender`, which was waiting for this signal (having already received the provisional acknowledgment from the leaseholder's evaluator).
+ it sends a commit acknowledgment to the gateway node's `DistSender`, which was waiting for this signal (having already received the provisional acknowledgment from the leaseholder's evaluator).
 1. The gateway node's `DistSender` aggregates commit acknowledgments from all of the write operations in the `BatchRequest`, as well as any values from read operations that should be returned to the client.
 1. Once all operations have successfully completed (i.e., reads have returned values and write intents have been committed), the `DistSender` tries to record the transaction's success in the transaction record (which provides a durable mechanism of tracking the transaction's state), which can cause a few situations to arise:
     - It checks the timestamp cache of the range where the first write occurred to see if its timestamp got pushed forward. If it did, the transaction performs a [read refresh](transaction-layer.html#read-refreshing) to see if any values it needed have been changed. If the read refresh is successful, the transaction can commit at the pushed timestamp. If the read refresh fails, the transaction must be restarted.
 	- If the transaction is in an `ABORTED` state, the `DistSender` sends a response indicating as much, which ends up back at the SQL interface.
 
-	Upon passing these checks the transaction record is either written for the first time with the `COMMITTED` state, or if it was in a `PENDING` state, it is moved to `COMMITTED`. Only at this point is the transaction considered committed.
+	Upon passing these checks the transaction record is either written for the first time with the `STAGING` state, or if it was in a `PENDING` or `STAGING` state, it is moved to `COMMITTED`. Only at this point is the transaction considered committed.
```

Comment:
I'm a little fuzzy on whether the update to this sentence is correct.  I tried to make a minimal edit to the existing content, but perhaps a larger change is needed.  I'm also trying hard not to duplicate content that is already expressed more canonically in #5394.

What do you think Nathan?

(PS I hope this is ignored until Monday - working a bit today because I missed time this week due to illness).